packages/components/src/systems/PharmacySelect/index.tsx

```diff
@@ -108,31 +106,12 @@ export default function PharmacySelect(props: PharmacySelectProps) {
       <div class="pt-4">
         <Show when={loadedTabs().includes('Send to Patient')}>
           <div class={tab() !== 'Send to Patient' ? 'hidden' : ''}>
-            {patientCount() > 0 && (
-              <RadioGroupCards
-                label="Patients"
-                initSelected={props?.patientIds?.[0]}
-                setSelected={(patientId) => {
-                  // for now, there is only one patient, but in the future we
-                  // might want to support selecting between multiple patients
-                  if (props.setPatientId) {
-                    props.setPatientId(patientId);
-                  }
-                  if (props.setPharmacyId) {
-                    props.setPharmacyId(undefined);
-                  }
-                }}
-              >
-                <For each={props?.patientIds || []}>
-                  {(id) => (
-                    <RadioGroupCards.Option value={id}>
-                      <PatientDetails patientId={id} />
-                    </RadioGroupCards.Option>
-                  )}
-                </For>
-              </RadioGroupCards>
-            )}
-            {patientCount() === 0 && <div>Please add a patient.</div>}
+            <Show
+              when={(props?.patientIds?.length || 0) > 0}
+              fallback={<div>Please select a patient.</div>}
+            >
+              <SendToPatient patientId={props.patientIds[0]} />
+            </Show>
```

Comment:
What happens if theres more than one patient here?

Reply:
We don't have this use case yet, this was anticipating multiple caregivers.